src/app.ts

```diff
@@ -9,9 +9,17 @@ const homeRoute = require('./api/routes/home')
 const usersRoute = require('./api/routes/users')
 const testRoute = require('./api/routes/usertest')
 
+const options = {
+  useNewUrlParser: true,
+  useCreateIndex: true,
+  useFindAndModify: false,
+};
 // Connect to database
-mongoose.connect(`${process.env.DB_CONNECTION_URL}`,
-    { useUnifiedTopology: true, useNewUrlParser: true })
+const url = `mongodb://${process.env.MONGO_NON_ROOT_USERNAME}:${process.env.MONGO_NON_ROOT_PASSWORD}@mongodb:27017/${process.env.MONGO_INITDB_DATABASE}`
```

Comment:
We should find a way to adjust the `url` such that both the docker and non-docker options can be used.